src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx

```diff
@@ -30,25 +32,45 @@ class PaymentInformationEditModal extends React.Component {
   onSubmit = event => {
     event.preventDefault();
 
-    const {
+    let {
       financialInstitutionRoutingNumber: routingNumber,
       accountNumber,
       accountType,
     } = this.props.fields;
 
-    const routingNumberErr = getRoutingNumberErrorMessage(
-      routingNumber.field.value,
+    routingNumber = set('field.dirty', true, routingNumber);
+    routingNumber = set(
+      'errorMessage',
+      getRoutingNumberErrorMessage(routingNumber.field.value),
+      routingNumber,
+    );
+
+    accountNumber = set('field.dirty', true, accountNumber);
+    accountNumber = set(
+      'errorMessage',
+      getAccountNumberErrorMessage(accountNumber.field.value),
+      accountNumber,
     );
-    const accountNumberErr = getAccountNumberErrorMessage(
-      accountNumber.field.value,
+
+    accountType = set('field.dirty', true, accountType);
+    accountType = set(
+      'errorMessage',
+      getAccountTypeErrorMessage(accountType.value.value),
+      accountType,
+    );
+
+    this.props.editModalFieldChanged(
+      'financialInstitutionRoutingNumber',
+      routingNumber,
     );
-    const accountTypeErr = getAccountTypeErrorMessage(accountType.value.value);
+    this.props.editModalFieldChanged('accountNumber', accountNumber);
+    this.props.editModalFieldChanged('accountType', accountType);
 
-    if (routingNumberErr) {
+    if (routingNumber.errorMessage) {
       focusElement('[name=routing-number]');
-    } else if (accountNumberErr) {
+    } else if (accountNumber.errorMessage) {
       focusElement('[name=account-number]');
-    } else if (accountTypeErr) {
+    } else if (accountType.errorMessage) {
```

Comment:
These changes ultimately execute the validators on all the form fields and display any/all errors. Previously, it was only showing at most one error during the submit.